cypress/integration/User Management/BL_00_ReceptionistEdit.js

```diff
@@ -0,0 +1,128 @@
+describe("Fills out form with mock data of receptionist from our user.json file, view our form to see if the data is inputed properly, edit our form with edited data, and submits our form", () => {
```

Comment:
You need to give this a high level description, so no "user.json" should be mentioned here. This should be named `describe("Editing an existing Receptionist account details")` 

These titles should be very simple to understand - even a 5 year old should be able to understand this